arm/Microsoft.Compute/virtualMachines/.parameters/linux.autmg.parameters.json

```diff
@@ -0,0 +1,64 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-vm-linux-autmg-01"
```

Comment:
```suggestion
            "value": "<<namePrefix>>-vm-linatmg-001"
```

Reply:
Linux vms do not have the same limitation